### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SoaringEarth**
Updated programming test to make our expectations clear

### CODE REVIEW

File: interview/task/README.md [hunk 1]
```diff
@@ -9,10 +11,22 @@ The cells of the list should display information about an event as follows:
...
+[http://my-json-server.typicode.com/livestyled/mock-api/events](http://my-json-server.typicode.com/livestyled/mock-api/events)
+
 *Note, this endpoint is paginated, and returns only 10 results at a time. To display the full list, you will either need to implement pagination by fetching the desired page using the \_page query parameter, or increase the limit on the response length using the limit query parameter. We'll leave it up to you to decide which of these we'd prefer* 😉.
 
 We would like you to allow the user to favourite/unfavourite events by tapping a button on the cell which indicates the current state. These actions should be persisted locally on the device.
 
+We would like to see a completed test containing the following: 
+- A readme
+- A compiling project
+- A scalable implementation of Pagination
```
Comment (rossrossp) [inline interview/task/README.md]: Pagination doesn't need capitalisation.
Also, do we expect juniors to do the pagination? If not, could we say "OR a comment explaining how you would approach it"?

File: interview/task/README.md [hunk 2]
```diff
@@ -9,10 +11,22 @@ The cells of the list should display information about an event as follows:
...
+- An implementation of Persistent Storage
```
Comment (rossrossp) [inline interview/task/README.md]: Persistent Storage doesn't need capitalisation

File: interview/task/README.md [hunk 3]
```diff
@@ -9,10 +11,22 @@ The cells of the list should display information about an event as follows:
...
+- Good project organisation
+- Good choice of architectural pattern (we don't mind which but think about how the test could be extended in the future)
```
Comment (rossrossp) [inline interview/task/README.md]: architectural pattern -> design pattern

File: interview/task/README.md [hunk 4]
```diff
@@ -9,10 +11,22 @@ The cells of the list should display information about an event as follows:
 ![Event Cells](event-cells.jpeg "Event Cells")
 
 Events should be fetched using the mock endpoint available at:
-[https://my-json-server.typicode.com/livestyled/mock-api/events](https://my-json-server.typicode.com/livestyled/mock-api/events).  
+[http://my-json-server.typicode.com/livestyled/mock-api/events?_page=1](http://my-json-server.typicode.com/livestyled/mock-api/events?_page=1).  
+
+You can see the full list of events here:
```
Comment (rossrossp) [inline interview/task/README.md]: Do we even need to provide the full list URL? I would say provide only the paginated URL...

File: interview/task/README.md (hunk 2 above)
Comment (mickeylee25) [inline interview/task/README.md]: Could change to 'An implemention of offline usage'

**Review by mickeylee25 — APPROVED** [review]

**Review by rossrossp — APPROVED** [review]

**Pull request merged**